### PR TITLE
DOCS-1426-Patch-3.15.3

### PR DESCRIPTION
**Pull request by bmckercher123**
CE 3.15.3. 

Product Version(s):
CE

Issue:
n/a

Link to docs preview:
https://deploy-preview-635--tigera.netlify.app/calico-enterprise/3.15/release-notes/

SME review:
- [x] An SME has approved this change. 

DOCS review:
- [x] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [x] Deploy preview inspected wherever changes were made 
- [x] Build completed successfully
- [ ] Tests passed. WAITING FOR ENG MERGE so build tests pass.

### CODE REVIEW

File: calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.3-release-notes.mdx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+08 May 2023
+
+### Improvements
+
+Egress gateways will attempt to auto detect the iptables backend, preferring `nft` if available.
```
Comment (matthewdupre) [inline calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.3-release-notes.mdx]: - Update rbac.authorization.k8s.io apiVersion to v1

File: calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.3-release-notes.mdx [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+### Bug Fixes
+
```
Comment (matthewdupre) [inline calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.3-release-notes.mdx:12]: - Update golang to 1.20.4 and update other dependencies to remove CVEs.

**Review by matthewdupre — APPROVED** [review]
Just a couple of lines for notes

File: calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.3-release-notes.mdx (hunk 1 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.3-release-notes.mdx]: Added.

File: calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.3-release-notes.mdx (hunk 2 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.3-release-notes.mdx:12]: Added.

**Pull request merged**